src/modules/keepalive/keepalive_rpc.c

```diff
@@ -87,3 +100,137 @@ static void keepalive_rpc_list(rpc_t *rpc, void *ctx)
 
 	return;
 }
+
+static void keepalive_rpc_add(rpc_t *rpc, void *ctx)
+{
+	str sip_adress = {0,0};
+	str table_name ={0,0};
+	int ret = 0;
+
+	ret = rpc->scan(ctx, "SS",&sip_adress,&table_name);
+
+	if (ret < 2) {
+		LM_ERR("not enough parameters - read so far: %d\n", ret);
```

Comment:
When reading the log file, you probably want to add the command name or some context 
LM_ERR("keepalived.add : not enough parameters - read so far: %d\n", ret);

Reply:
I haven't changed this one. The C function name is printed in each log message, being suggestive enough, but if you want to make an update, you can push it to the repo.